Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocTextValue.css

```diff
@@ -21,6 +21,8 @@ span.bocTextValue textarea
   display: inline-flex;
   align-items: center;
   width: fit-content;
+  overflow-wrap: anywhere;
+  word-break: normal;
```

Comment:
I recommend moving this down to line 37-ish, to the section where the input+span / textarea+span styles are set. Makes it more consistent with the rest of the styles